### ISSUE TITLE
new contract not sync on wallet with v 0.9.1 and light client

### ISSUE THREAD

**Issue by stbregeon** [ethereum/mist#3111]
v 0.9.1
geth 1.7.0
Sync with light client beta

windows 8.1

Hello,

I create a new contrat today with v 0.9.1

It's ok on etherscan in block 4 330 447 (Contract Address 0x4d83E56Ab2d905ADDa29ce77bFa190a9fF42C7bf)

But I can't see it on my account overwier 
(so I can't handle it)

does Someone know waht happens ??

best regards

**Comment by vess28** [comment]
I have the same issue using Windows 10.
Any help is appreciated!

**Comment by hellodeng** [comment]
Don't use this verion (v0.9.1) any more. This is deadly bad version, which can't send TOKEN/ETHER at all. I've tested it for tow days. Please someone help us out of this.

**Comment by will2245** [comment]
I am having the same issue in Windows 7 64 bit. I did confirmed wallet contract was created on the blockchain (I check using https://etherscan.io) but the wallet still appears to be creating in Mist.  I think this is related to an the Uncaught Type Error: Cannot read properly 'toLowerCase' of undefined I see in the developer tool console.  See the attached picture for details.

![mist 0 91 uncaught type error](https://user-images.githubusercontent.com/30986776/31326495-d1c15780-ac95-11e7-9e67-d026adc7cd23.png)

Is there some way for me to fix this issue and have my wallet contract accessible or is this an bug that will be fixed in a later version of the program?

**Comment by 420TOKEJ** [comment]
I am having a similar problem windows 10 64bit geth 1.7.0 and the new version 0.9.1 I tried to create a contract and a new wallet. I can send and receive ether from my main account but watched tokens like unicorns and previous contracts admin pages are black and white instead of colorful. the node is fully synced and can send and receive ether but certain aspects of the contracts and wallets seem out of sync. I deployed a contract and can see it on etherscan and access it through myetherwallet.
